Example/src/screens/SimpleNativeStack.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import {View, StyleSheet} from 'react-native';
+import {enableScreens} from 'react-native-screens';
+import {
+  createNativeStackNavigator,
+  NativeStackNavigationProp,
+} from 'react-native-screens/native-stack';
+import {Button} from '../shared';
+
+enableScreens();
+
+type StackParamList = {
+  Main: undefined;
+  Detail: undefined;
+};
+
+interface MainScreenProps {
+  navigation: NativeStackNavigationProp<StackParamList, 'Main'>;
+}
+
+const MainScreen = ({navigation}: MainScreenProps): JSX.Element => (
+  <View style={styles.container}>
+    <Button
+      title="Go to detail"
+      onPress={() => navigation.navigate('Detail')}
+    />
+    <Button onPress={() => navigation.pop()} title="🔙 Back to Examples" />
+  </View>
+);
+
+interface DetailScreenProps {
+  navigation: NativeStackNavigationProp<StackParamList, 'Detail'>;
+}
+
+const DetailScreen = ({navigation}: DetailScreenProps): JSX.Element => (
+  <View style={styles.container}>
+    <Button title="Go back" onPress={() => navigation.goBack()} />
+  </View>
+);
+
+const Stack = createNativeStackNavigator<StackParamList>();
+
+const App = (): JSX.Element => (
+  <Stack.Navigator
+    screenOptions={{
+      statusBarStyle: 'dark',
```

Comment:
same

Reply:
Fixed in https://github.com/software-mansion/react-native-screens/pull/827/commits/85e8e17b756652507b71956a30319822422c5358